staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1982,3 +1989,19 @@ enum CgroupDriver {
     CGROUPFS = 1;
 }
 
+message UpdatePodSandboxResourcesRequest {
+    // ID of the PodSandbox to update.
+    string pod_sandbox_id = 1;
+
+    // Optional overhead represents the overheads associated with this sandbox
+    LinuxContainerResources overhead = 2;
+    // Optional resources represents the sum of container resources for this sandbox
+    LinuxContainerResources resources = 3;
+
+    // Unstructured key-value map holding arbitrary additional information for
+    // sandbox resources updating. This can be used for specifying experimental
+    // resources to update or other options to use when updating the sandbox.
+    map<string, string> annotations = 4;
```

Comment:
Per [comment here](https://github.com/containerd/nri/pull/141#issuecomment-2654961418), I think we should remove this field for now.